src/module/classStorage.js

```diff
@@ -48,6 +48,24 @@ class Storage {
     });
     localStorage.setItem('tasks', JSON.stringify(tasks));
   }
+
+  static editCompletion(code, completed) {
+    const tasks = Storage.getTasks();
+    tasks.forEach((task) => {
+      if (task.code === parseInt(code, 10) && (completed)) {
+        task.completed = true;
+      } else if (task.code === parseInt(code, 10) && (!completed)) {
+        task.completed = false;
+      }
+    });
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static clearAllCompleted() {
+    const tasks = Storage.getTasks();
+    const completedDeleted = tasks.filter((item) => item.completed === false);
+    localStorage.setItem('tasks', JSON.stringify(completedDeleted));
+  }
```

Comment:
Well done on implementing the changes required by the previous reviewer. The localStorage is updated upon interacting with the to-do list checkboxes. 👏🏾👍🏾 

However, after the page reload if there were checkboxes that were checked they need to remain checked. In the example below, I have two tasks that were checked before reloading. After reloading the boxes are now unchecked even though the property `completed` remain true in the local storage array.

![image](https://user-images.githubusercontent.com/38088818/213728423-fbfffd1c-6790-4091-a800-d9b39766aabf.png)

- Kindly make sure that your data is consistent with the UI.

Reply:
Hello, @KDlamini I only have one review available I need to be clear on this, at this point when I click the reload button I clear all the checked tasks from the array, the requirement is to keep the checked tasks after the reload and only erase them with the clear all completed button right?

Reply:
Hey, @gealsanchez try refreshing with the browser refresh button you'll see what I'm trying to demonstrate. 

- Try populating the to-do list data on the page the same way you would when the application loads.

Reply:
Ok thats fine and when the reload is clicked do I have to erase all the checked tasks or just reload?

Reply:
@gealsanchez if by reload you mean the `browser refresh` then nothing should be erased. All the tasks should remain the same. Checked boxes should stay checked after refresh.